More info for Crowdin usage (#1130)
akeneo/pim-docs: contribute_to_pim/translate.rst

.. image:: ./crowdin-logo.png

How to translate?
-----------------

> You have to to create an account and begin to translate in your favorite language on `Crowdin`_.

Reviewer: You have to ~to~ create

> You have to to create an account and begin to translate in your favorite language on `Crowdin`_.

Author: ```suggestion
You have to create an account and begin to translate in your favorite language on `Crowdin`_.
```

You need a proofreading permission to validate the strings, so your translation will be included in the PIM.
Each language with more than 80% of valid translations are automatically translated with `Nelson`_ each week, and will be available in the next patch.
You will be able to claim your badge "El Translator" on `Badger platform <http://badger.akeneo.com/login/>`_!

Your language is not available on Crowdin?
------------------------------------------

> As Akeneo PIM is Open Source, we will be glad to add welcome new languages for our application.

Author: ```suggestion
As Akeneo PIM is Open Source, we will be glad to welcome new languages for our application.
```

> As Akeneo PIM is Open Source, we will be glad to add welcome new languages for our application.

Reviewer: we will be glad to ~add~ welcome

Don't hesitate to ask to a `Crowdin`_ administrator to add your language to the available languages list of Crowdin.

> Please reminder that only languages validated in more than 80% will be synchronized in Akeneo PIM.

Author: ```suggestion
We remind you that only languages validated in more than 80% will be synchronized in Akeneo PIM.
```

> Please reminder that only languages validated in more than 80% will be synchronized in Akeneo PIM.

Reviewer: ~Please reminder~ We remind you that

> You want to use a non fully translated language in your application?

Author: ```suggestion
How to use a partially translated language in your application?
```

> You want to use a non fully translated language in your application?

Reviewer: How to use a partially translated language in your application?

> --------------------------------------------------------------------

Author: ```suggestion
---------------------------------------------------------------
```

.. note::

>     We strongly encourage you to use the default way, that is validate the application in more than 80% and wait for the next patch.

Author: ```suggestion
    We strongly encourage you to use the default way: validate translations in more than 80% and wait for the next patch.
```

>     We strongly encourage you to use the default way, that is validate the application in more than 80% and wait for the next patch.

Reviewer: I don't understand this sentence

>     Indeed, an application not fully translated will contain a lot of non translated strings in many places of your application and can lead to a very bad user experience.

Author: ```suggestion
    Indeed, a partially translated PIM will contain a lot of non translated strings in many places of your application and can lead to a very bad user experience.
```

>     Indeed, an application not fully translated will contain a lot of non translated strings in many places of your application and can result in a very bad user experience.

Author: ```suggestion
    Indeed, an application not fully translated will contain a lot of non translated strings in many places of your application and can lead to a very bad user experience.
```

>     Indeed, an application not fully translated will contains a lot of non translated strings in many places of your application and can result of a very bad user experience.

Reviewer 2: ```suggestion
    Indeed, an application not fully translated will contain a lot of non translated strings in many places of your application and can result in a very bad user experience.
```

>     Indeed, an application not fully translated will contains a lot of non translated strings in many places of your application and can result of a very bad user experience.

Reviewer: "Indeed, a partially translated PIM can lead to a very bad user experience."

> By default, Akeneo PIM only displays and synchronize languages validated in more than 80%.

Reviewer 2: ```suggestion
By default, Akeneo PIM only displays and synchronizes languages validated in more than 80%.
```
Also, " in more than 80%" seems weird, any English reviewer to suggest a better thing?

> By default, Akeneo PIM only displays and synchronize languages validated in more than 80%.

Reviewer: synchronizes

If you have access to the code of your application, you can however set your user interface with another language:

> Start by downloading the complete archive of all the managed translation in Crowdin `here <https://crowdin.com/backend/download/project/akeneo.zip>`_ and extract it.

Reviewer 2: ```suggestion
Start by downloading the complete archive of all the managed translations in Crowdin `here <https://crowdin.com/backend/download/project/akeneo.zip>`_ and extract it.
```

> Start by downloading the complete archive of all the managed translation in Crowdin `here <https://crowdin.com/backend/download/project/akeneo.zip>`_ and extract it.

Reviewer: translations

> Then, copy the translations in the right folders. For example, if you are using 3.0 version:

Reviewer: in the correct folder depending on the version you use?

.. code-block:: bash

    cp -r crowdin_archive/3.0/Community/* your_app/vendor/akeneo/pim-community-dev/

> If you are in Enterprise Edition, run this command too:

Author: ```suggestion
If you use the Enterprise Edition, run this command too:
```

> If you are in Enterprise Edition, run this command too:

Reviewer: If you use the
